dashboard/src/components/Table/TreeTable.tsx

```diff
@@ -23,8 +25,18 @@ const treeTableColumnsLabelId = [
 const TreeTableRow = (row: TreeTableBody): JSX.Element => {
   const backgroundClassName =
     'flex flex-row bg-lightGray w-fit h-fit p-1 rounded-lg';
+
+  const navigate = useNavigate();
+
+  const navigateToTreeDetailPage = useCallback(
+    (treeId: string) => {
+      navigate(`tree/${treeId}`);
+    },
+    [navigate],
+  );
```

Comment:
```suggestion
  const navigateToTreeDetailPage = useCallback(
    () => {
      navigate(`tree/${row.id}`);
    },
    [navigate],
  );
```

so you can take advantage of memoization and use `<TableRow onClick={navigateToTreeDetailPage}>`